sys-apps/policycoreutils/policycoreutils-3.6-r1.ebuild

```diff
@@ -0,0 +1,210 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_REQ_USE="xml(+)"
+
+inherit python-r1 toolchain-funcs bash-completion-r1
+
+MY_PV="${PV//_/-}"
+MY_P="${PN}-${MY_PV}"
+EXTRAS_VER="1.37"
+
+DESCRIPTION="SELinux core utilities"
+HOMEPAGE="https://github.com/SELinuxProject/selinux/wiki"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/SELinuxProject/selinux.git"
+	SRC_URI="!vanilla? ( https://dev.gentoo.org/~perfinion/distfiles/policycoreutils-extra-${EXTRAS_VER}.tar.bz2 )"
+	S1="${WORKDIR}/${P}/${PN}"
+	S2="${WORKDIR}/policycoreutils-extra"
+	S="${S1}"
+else
+	SRC_URI="https://github.com/SELinuxProject/selinux/releases/download/${MY_PV}/${MY_P}.tar.gz
+		!vanilla? ( https://dev.gentoo.org/~perfinion/distfiles/policycoreutils-extra-${EXTRAS_VER}.tar.bz2 )"
+	KEYWORDS="~amd64 ~arm ~arm64 ~mips ~x86"
+	S1="${WORKDIR}/${MY_P}"
+	S2="${WORKDIR}/policycoreutils-extra"
+	S="${S1}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="audit pam +python split-usr vanilla"
+REQUIRED_USE="
+	!vanilla? ( python ${PYTHON_REQUIRED_USE} )
+"
+
+DEPEND="
+	python? (
+		>=sys-libs/libselinux-${PV}:=[python,${PYTHON_USEDEP}]
+		>=sys-libs/libsemanage-${PV}:=[python(+),${PYTHON_USEDEP}]
+		audit? ( >=sys-process/audit-1.5.1[python,${PYTHON_USEDEP}] )
+		${PYTHON_DEPS}
+	)
+	!python? (
```

Comment:
I need need to look up some things again, pretty sure there is an easier way to do this tho.